src/app/zap-templates/zcl/data-model/chip/software-diagnostics-cluster.xml

```diff
@@ -45,7 +39,9 @@ limitations under the License.
     </command>
     <event side="server" code="0x00" name="SoftwareFault" priority="info" optional="true">
       <description>Indicate the last software fault that has taken place on the Node.</description>
-      <field id="0" name="SoftwareFault" type="SoftwareFaultStruct"/>
+      <field id="0" name="Id" type="INT64U"/>
+      <field id="1" name="Name" type="CHAR_STRING" length="8"/>
+      <field id="2" name="FaultRecording" type="OCTET_STRING" length="1024"/>
```

Comment:
These two fields need to be optional, no?  The PR descroption says they should be... but they're not here.